src/Sarif.Converters/FortifyFprConverter.cs

```diff
@@ -0,0 +1,631 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Text;
+using System.Xml;
+
+namespace Microsoft.CodeAnalysis.Sarif.Converters
+{
+    internal class FortifyFprConverter : ToolFileConverterBase
+    {
+        private const string FortifyToolName = "HP Fortify Static Code Analyzer";
+        private const string FortifyExecutable = "[REMOVED]insourceanalyzer.exe";
+
+        private readonly NameTable _nameTable;
+        private readonly FortifyFprStrings _strings;
+
+        private XmlReader _reader;
+        private Invocation _invocation;
+        private string _runId;
+        private string _automationId;
+        private List<Result> _results = new List<Result>();
```

Comment:
> = new List<Result>() [](start = 37, length = 21)

can we put all this instance member initialization in the constructor. I prefer this mostly so that debugging doesn't jump outside of methods.